src/applications/caregivers/tests/unit/components/FormFields/FacilitySearch.unit.spec.js

```diff
@@ -405,19 +408,83 @@ describe('CG <FacilitySearch>', () => {
           expect(selectors().loader).to.not.exist;
           expect(selectors().input).to.not.have.attr('error');
           expect(getByText(/Showing 1-1 of 1 facilities for/)).to.exist;
+          expect(selectors().ariaLiveStatus.textContent).to.eq('');
         });
 
         await waitFor(() => {
           userEvent.click(selectors().moreFacilities);
           expect(selectors().radioList).to.exist;
           expect(selectors().loader).to.exist;
+          expect(selectors().ariaLiveStatus.textContent).to.eq('');
         });
 
         await waitFor(() => {
           expect(selectors().radioList).to.exist;
           expect(selectors().loader).to.not.exist;
           expect(getByText(/Showing 1-2 of 2 facilities for/)).to.exist;
           expect(selectors().input).to.not.have.attr('error');
+          expect(selectors().ariaLiveStatus.textContent).to.eq(
+            '1 new facility loaded',
+          );
+        });
+
+        await waitFor(() => {
+          sinon.assert.calledWith(mapboxStub, 'Tampa');
+          sinon.assert.calledWith(facilitiesStub, {
+            lat,
+            long,
+            perPage,
+            radius,
+            page: 2,
+          });
+          expect(mapboxStub.callCount).to.equal(1);
+          // This is 3 because there is an existing bug that using submit with the va-search-input component triggers two requests
+          // https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3117
+          expect(facilitiesStub.callCount).to.equal(3);
+        });
+      });
+
+      it('successfully loads 2 more facilities on click', async () => {
```

Comment:
I ended up somewhat duplicating the previous test for the different facility message texts that can show up. I started a refactor to make a reusable helper since a lot of the `Show More Facilities` tests are somewhat similar, but I wasn't liking how it was turning out so I kept it more verbose like this. I'm open to looking further into this if someone really thinks we should, but also fine to leave it as is.